### ISSUE TITLE
DataTable: Expander with grouping

### ISSUE THREAD

**Issue by nedimcimpo** [primefaces/primereact#4905]
### Describe the bug

I wanted to create table where there would be grouping by 2. Let's say if in case of working with months it would be like first 2 weeks and last two weeks. Inside those I wanted to be able to expande that row for first two weeks where I would have days or smetginh else. But... Since in this example below I have group of two and that td has rowspan of 2, as soon as I do the expand goes off becase rowspan stays the same and now we have 3 insted of 2 rows or maybe even 4 if we expand both first and second week. I've tried to manualy set number of rowspan for that column but it's not working as expected. Is there a solution or workaround?

### Reproducer

https://codesandbox.io/s/primereact-demo-forked-fsljr7?file=/src/App.tsx

### PrimeReact version

9.5.0

### React version

18.x

### Language

TypeScript

### Build / Runtime

Create React App (CRA)

### Browser(s)

Chrome

**Comment by melloware** [comment]
Are either one of these issues your issue?

https://github.com/primefaces/primereact/issues/2598

https://github.com/primefaces/primereact/issues/619

**Comment by nedimcimpo** [comment]
> Are either one of these issues your issue?
> 
> #2598
> 
> #619

I don't think so. Second one is close. The thing is that when I do the grouping, I want to be able to expand row in that group and that group rowspan goes up by the amount of rows that have been expanded in that group. Rowspan for that grouped cell stays the same and its causing issues.

**Comment by melloware** [comment]
OK thanks for checking.
